numpy/core/src/multiarray/sequence.c

```diff
@@ -47,6 +47,9 @@ array_contains(PyArrayObject *self, PyObject *el)
     }
 
     ret = PyObject_IsTrue(any);
+    if (ret == -1) {
+       return -1;
+    }
```

Comment:
This addition is unnecessary and wrong, since it side-steps the cleanup of `any` and otherwise does nothing.